packages/server/migrations/1550779717609_full-db-migration.js

```diff
@@ -0,0 +1,94 @@
+exports.up = pgm => {
+  //1. Users Table
+  pgm.sql(`
+    CREATE TABLE "foostown"."users" (
+      "id" SERIAL PRIMARY KEY,
+      "fullname" TEXT UNIQUE NOT NULL,
+      "email" TEXT UNIQUE NOT NULL,
+      "password" TEXT NOT NULL
+    );
+  `),
+  //2. Teams Table
+  pgm.sql(`
+    CREATE TABLE "foostown"."teams" (
+      "id" SERIAL PRIMARY KEY,
+      "team_name" TEXT NOT NULL,
+      "organization_id" TEXT NOT NULL,
+      UNIQUE(team_name, organization_id)
+    );
+  `),
+  //3. Organizations Table
+  pgm.sql(`
+    CREATE TABLE "foostown"."organizations" (
+      "id" SERIAL PRIMARY key,
+      "name" text UNIQUE NOT NULL,
+      "is_active" BOOLEAN NOT NULL,
+      "owner_id" INTEGER REFERENCES users(id) NOT NULL
+    );
+  `),
+  //4. Organizations_Users Table
+  pgm.sql(`
+    CREATE TABLE "foostown"."organizations_users" (
+      "organization_id" INTEGER  REFERENCES organizations (id) NOT NULL,
```

Comment:
Extra space between INTEGER and REFERENCES

Reply:
fixed